app/src/main/java/com/devshish/internship/presentation/service/player/MediaSessionCallback.kt

```diff
@@ -69,17 +88,6 @@ class MediaSessionCallback(
         Timber.d("onPause")
     }
 
-    override fun onStop() {
-        super.onStop()
-        exoPlayer.stop()
-        mediaSession.updateState(
-            stateBuilder = stateBuilder,
-            state = PlaybackStateCompat.STATE_STOPPED
-        )
-        mediaService.stopSelf()
-        Timber.d("onStop")
-    }
-
```

Comment:
question just for clarification: why you removed this code? (you can write here or in telegram)